packages/slate-react/src/components/editable.tsx

```diff
@@ -471,6 +499,39 @@ export const Editable = (props: EditableProps) => {
   // https://github.com/facebook/react/issues/11211
   const onDOMBeforeInput = useCallback(
     (event: InputEvent) => {
+      const el = ReactEditor.toDOMNode(editor, editor)
+      const root = el.getRootNode()
+      const safariVersion = navigator.userAgent.match(/Version\/(\d+\.\d+)/)
+      const isSafariVersionLessThan17 =
+        IS_WEBKIT && safariVersion && parseFloat(safariVersion[1]) < 17.0
+
+      if (
+        isSafariVersionLessThan17 &&
+        processing?.current &&
+        IS_WEBKIT &&
+        root instanceof ShadowRoot
+      ) {
+        // @ts-ignore
```

Comment:
why is ignore needed here?

Reply:
leftover [aded41d](https://github.com/ianstormtaylor/slate/pull/5648/commits/aded41dde434fb838c1ecbf0d5dd55c36b810299)